packages/create/examples/graphql-modules/src/app/common/common.module.ts

```diff
@@ -3,6 +3,6 @@ import { PrismaProvider } from './Prisma.provider';
 import { sdlInputs } from '@paljs/plugins';
 
 export const CommonModule = new GraphQLModule({
-  typeDefs: sdlInputs(),
+  typeDefs: sdlInputs({}),
```

Comment:
Fix this issue not here but in the `sdlInputs` function place to make args optional 
 here https://github.com/paljs/prisma-tools/blob/f3f35b57b5c9fac4cfd4b4361feddca5897ef04a/packages/plugins/src/sdlInputs.ts#L103